widget/info.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "recordSummaryTile",
     "title": "Record Summary Tile",
-    "subTitle": "Widget lists, number of customers and Alerts",
+    "subTitle": "The Record Summary widget extracts JSON data from your FortiSOAR module records and presents it in a convenient tile format.",
```

Comment:
This widget renders JSON record from a FortiSOAR module and presents it in a convenient tile format.
What is convenient means here?

Reply:
Reviewed and changed